addons/resource.language.en_gb/resources/strings.po

```diff
@@ -1581,7 +1581,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#348"
-msgid "Enable passthrough"
+msgid "Allow passthrough"
```

Comment:
What about `Enable passthrough support` or `Passthrough capable device`. Might make it more clear that it's not an always on thingy. `Allow passthrough` sounds a bit odd to me - not sure how it sounds to native speakers.